core/src/main/resources/hudson/tasks/ArtifactArchiver/config.jelly

```diff
@@ -28,6 +28,9 @@ THE SOFTWARE.
 	  <f:textbox />
 	</f:entry>
   <f:advanced>
+    <f:entry title="${%Base Path}" field="basePath">
```

Comment:
Please display only if `it instanceof AbstractProject`. This option is useless noise for Pipeline, since you can already

```groovy
dir('whatever') {
  archiveArtifacts 'right-here.txt'
}
```